# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderImageHost.h
# Commit: ce28584e3869
# Full Hash: ce28584e386976fc6df815260cdfebd8e6ffbadd
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-12-19 05:06:09
# Regressor Bug: 1829026
# File Overlap Count: 1
# Description:
#   Bug 1829026 - Update CanvasTranslator to work with DrawTargetWebgl. r=aosmond
#   
#   This adds the necessary infrastructure for CanvasTranslator to allocate DrawTargetWebgl
#   instead of just allocating TextureData, and to use RemoteTextureMap to handle sending
#   the DrawTargetWebgl frames to the compositor.
# ==============================================================================

diff -r bb4029ea69bb -r ce28584e3869 gfx/layers/wr/WebRenderImageHost.h
--- a/gfx/layers/wr/WebRenderImageHost.h	Mon Dec 18 18:10:46 2023 +0000
+++ b/gfx/layers/wr/WebRenderImageHost.h	Mon Dec 18 18:10:46 2023 +0000
@@ -88,6 +88,7 @@
 
   std::deque<CompositableTextureHostRef> mPendingRemoteTextureWrappers;
   bool mWaitingReadyCallback = false;
+  bool mWaitForRemoteTextureOwner = true;
 
   Maybe<RemoteTextureOwnerId> mRemoteTextureOwnerIdOfPushCallback;
   base::ProcessId mForPidOfPushCallback;